dashboard/src/app/index.module.js

```diff
@@ -96,8 +96,51 @@ initModule.factory('AuthInterceptor', ($window, $cookies, $q, $location, $log) =
   };
 });
 
-initModule.config(['$routeProvider', '$locationProvider', '$httpProvider', ($routeProvider, $locationProvider, $httpProvider) => {
+initModule.factory('AddMachineTokenToUrlInterceptor', ($injector, $q) => {
```

Comment:
For me it looks quite similar to [this](https://github.com/codenvy/codenvy/blob/master/dashboard/src/app/index.module.js#L65-L97), why it's not separated too?